src/lib/drivers/extension.js

```diff
@@ -145,7 +145,7 @@ class ExtensionDriver extends Driver {
     return new Promise((resolve, reject) => {
       chrome.tabs.query(currentTab, tabs => {
         if (chrome.runtime.lastError) {
-          return reject(chrome.runtime.lastError);
+          return reject(chrome.runtime.lastError.message);
```

Comment:
> It expects to be able to pull the message property.

oh. this will actually fix a separate bug where an exception in the promise was thrown but it didnt get shown in the popup because `err.message` didnt exist. score.